test/helper.js

```diff
@@ -53,7 +53,9 @@ mocha.before(async () => {
     clients: defaultClients,
   });
 
-  webServer = await bin.server('test');
+  webServer = await bin.server('test', true);
+  // Ugh, again pretty ugly
+  exports.influx = bin.server.influx;
```

Comment:
Why not take it from `webServer` which you can add the property to in `server.js`.